MIL/C02_Basics/S01_Calculating.lean

```diff
@@ -351,12 +351,44 @@ example (a b : ℝ) : (a + b) * (a - b) = a ^ 2 - b ^ 2 := by
 #check mul_sub a b c
 #check add_mul a b c
 #check add_sub a b c
+#check sub_add a b c
+#check sub_self a
```

Comment:
If these are unnecessary, it would be nice to still get the second half merged (with the edits that prove they're unnecessary :)